### DISCUSSION TITLE
Caption Color?

### DISCUSSION THREAD

**Discussion by harrismail** [Open-Shell/Open-Shell-Menu#1639] [General]
Hi all, I've just installed 4.4.190 on a Windows 11 machine and (for me) this is excellent ;-)
One tiny query... Can the Menu "caption" background color (behind the windows name at the side of the menu) be changed as it seems to default to blue?
TIA Steve

**Comment by ge0rdi** [comment]
Caption background is part of the skin.

**Reply by harrismail** [reply]
Aaaah, OK - no problem.

**Comment by harrismail** [comment]
One last thing.... Is there a way to turn off the "Start" balloon tip when you hover over the start button?

**Reply by ge0rdi** [reply]
Nope.
On Windows 11 we are currently quite limited as it uses different technology for taskbar (XAML).
Many taskbar customizations don't work anymore.

**Reply by harrismail** [reply]
No problem - thanks again.
